scrapi/harvesters/crossref.py

```diff
@@ -13,27 +13,26 @@
 
 from six.moves import xrange
 from nameparser import HumanName
-from dateutil.parser import parse
 
 from scrapi import requests
 from scrapi import settings
 from scrapi.base import JSONHarvester
 from scrapi.linter.document import RawDocument
-from scrapi.base.helpers import build_properties, compose
+from scrapi.base.helpers import build_properties, compose, date_formatter
 
 logger = logging.getLogger(__name__)
 
 
 def process_contributor(author, orcid):
     name = HumanName(author)
-    return {
+    ret = {
         'name': author,
         'givenName': name.first,
         'additionalName': name.middle,
         'familyName': name.last,
-        'email': '',
-        'sameAs': [orcid or '']
+        'sameAs': [orcid] if orcid else []
```

Comment:
Empty list, indicating that nothing is in the sameAs array. Since this is required, it won't be stripped out by the cleaner.